lib/webpacker/webpack_runner.rb

```diff
@@ -12,9 +12,8 @@ def run
         ["yarn", "webpack"]
       end
 
-      if ARGV.include?("--debug")
+      if @argv.include?("--debug-webpacker")
```

Comment:
Does webpack complains, if an unsupported option is present? We might need to delete the custom ones then?